apps/namadillo/src/utils/index.ts

```diff
@@ -85,12 +85,18 @@ export const isNamadaAsset = (asset: Asset): boolean =>
 
 export const toDisplayAmount = (
   asset: Asset,
-  baseAmount: BigNumber
+  baseAmount: BigNumber,
+  skipNamada = false
```

Comment:
This smells, are we using this somewhere else? If yes, should we add this condition on this place instead of here?

Looks like we are passing a `displayAmount` to this fn, like `toDisplayAmount(displayAmount)` instead of `toDisplayAmount(baseAmount)`